xds/src/test/java/io/grpc/xds/orca/OrcaServiceImplTest.java

```diff
@@ -169,17 +167,17 @@ public void testRequestIntervalGreater() {
     call.request(1);
     OrcaLoadReport expect = OrcaLoadReport.newBuilder().putUtilization("buffer", 0.2).build();
     verify(listener).onMessage(eq(expect));
-    reset(listener);
```

Comment:
While I'm not wild about reset, this is a really silly reason to rewrite the code. The lint checker is just broken, and I think we should ignore it or file a bug.

Reply:
yea the checker seem wrong but just took the opportunity to make a little improvement, if we can call it.